packages/components/tooltip/stories/Tooltip.stories.tsx

```diff
@@ -212,3 +213,28 @@ export const WithDelays = () => {
     </>
   );
 };
+
+export const WithReactElement = () => {
+  return (
+    <>
+      <SectionHeading as="h3" marginBottom="spacingS">
+        With React Elements as children
+      </SectionHeading>
+
+      <Flex marginBottom="spacingS">
+        <Tooltip
```

Comment:
💡 I think the example would be better without the delay and isVisible properties